Userland/Demos/Mandelbrot/Mandelbrot.cpp

```diff
@@ -185,8 +221,8 @@ class MandelbrotSet {
     {
         auto y_mid = m_y_start + (m_y_end - m_y_start) / 2;
         auto aspect_corrected_y_length = (m_x_end - m_x_start) * m_bitmap->height() / m_bitmap->width();
-        m_y_start = y_mid - aspect_corrected_y_length / 2;
-        m_y_end = y_mid + aspect_corrected_y_length / 2;
+        m_y_start = y_mid + aspect_corrected_y_length / 2;
+        m_y_end = y_mid - aspect_corrected_y_length / 2;
```

Comment:
This looks like it's only caused by you inverting the sign of `y_start` and `y_end`